grafana/dashboards/cmode/qtree.json

```diff
@@ -72,14 +72,31 @@
     }
   ],
   "panels": [
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "gridPos": {
+        "h": 4,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 131,
+      "options": {
+        "content": "This dashboard requires the Qtree templates. These actions are required to use this dashboard: <br>1. If you are using the ZapiPerf collector, then you must enable the Qtree counters in `$HARVEST/conf/zapiperf/default.yaml`. <br>2. If you are using the RestPerf collector, then you must enable the Qtree counters in `$HARVEST/conf/restperf/default.yaml`.<br> Please see [qtree dashboard](https://github.com/NetApp/harvest/wiki/FAQ#qtree-dashboard) for more details. Enabling `qtree.yaml` may slow down data collection.",
```

Comment:
This dashboard requires enabling a qtree template. <br> Please see [qtree dashboard](https://github.com/NetApp/harvest/wiki/FAQ#qtree-dashboard) for more details. Enabling `qtree.yaml` may slow down data collection.